sr_robot_launch/launch/sr_bimanual_hardware_control_loop.launch

```diff
@@ -188,4 +188,14 @@
     <node name="arm_group_position_controller_spawner" pkg="controller_manager" type="spawner" output="screen"
           args="ra_trajectory_controller la_trajectory_controller"/>
   </group>
+
+  <arg name="record_hand_data" value="$(optenv SR_RECORD_HAND_DATA disabled)"/>
```

Comment:
Is this part of migration or a completely separate issue?

Reply:
its from Tom's change as I branched out before this merge